src/traces/scattermapbox/constants.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+// Must use one of the following fonts as the family, else default to 'Open Sans Regular'
+// See https://github.com/openmaptiles/fonts/blob/gh-pages/fontstacks.json
+var supportedFonts = [
```

Comment:
@archmoj Does Mapbox make their list of supported fonts public anywhere as part of their package? Just wondering if there's a way we can ensure this list is always up-to-date without updating it manually.

Reply:
No in fact we use latest v1 version of `mapbox-gl`. Due to the licence changes of their v2+ no significant v1 is expected. On the other hand we may switch to `maplibre-gl-js` which is the continuation of `mapb0x-gl` v1. See #5578.
BTW we could possibly improve our docs for these fonts in a separate PR.

Reply:
OK got it. Do you know to what extent `maplibre` supports different font weights? Basically, will adding these options make switching to `maplibre` significantly more complicated?

Reply:
@emilykl , there is a migration branch in here maplibre where we should be able to try out:
- #7015